### PR TITLE
Update mix.exs, config, and tests for ecto 0.14 support

### PR DESCRIPTION
**Pull request by schainks**
Update mix.exs ecto, postgrex, and modify config.exs to use ecto's new Sandboxed connection pool.  Also modified `confirmation_needed_changeset` test to pass a dummy user instead of empty changeset to get it to pass properly (not sure if that was "best" decision, though ;))

### CODE REVIEW

File: config/config.exs [hunk 1]
```diff
@@ -2,6 +2,8 @@
 # and its dependencies with the aid of the Mix.Config module.
 use Mix.Config
 
+config :my_app, Repo,
+  pool: Ecto.Adapters.SQL.Sandbox
```
Comment (manukall) [inline config/config.exs]: i don't think this should be in here. `config/test.exs` should be enough. the rest looks good, though.

**Comment by manukall** [comment]
great! if you remove that one thing from config.exs, I'll merge. thanks!!

**Comment by schainks** [comment]
Nice catch, thanks.  Done!

**Comment by manukall** [comment]
thanks again. i just published v 0.2.0 including your changes.
